resource/sass/components/mixins/branch.scss

```diff
@@ -0,0 +1,16 @@
+@mixin create-branch($branch-icon-width:$default-branch-icon-width, $branch-icon-height:$default-branch-icon-height) {
+    display: inline-block;
+    border: 1px solid $default-color-branch-border;
+    margin-left: 30px;
+    margin-top: 10px;
+    padding: 5px;
+    &__icon {
+        height: $branch-icon-height;
+        width: $branch-icon-width;
+    }
+    &__name {
+        font-family: $default-font-family;
+        font-family: $default-branch-font-size;
```

Comment:
somewhere we have color-branch, and some others we have branch-font-size. multiple naming convention in the same application is not good